platform_versioned_docs/version-24.1.0/compute-envs/hpc.mdx

```diff
@@ -21,37 +21,53 @@ To launch pipelines into an **HPC** cluster from Seqera, the following requireme
 
+## Credentials
+
+Seqera requires SSH access to your HPC cluster to run pipelines. Use [managed identities](../credentials/managed_identities.mdx) to enable granular access control and preserve individual cluster user identities. If you do not need to grant multiple users access to your cluster, use workspace [SSH credentials](../credentials/ssh_credentials.mdx) for cluster login. 
+
+For HPC clusters that do not allow direct access through an SSH client, a secure connection can be authenticated with [Tower Agent](../supported_software/agent/agent.mdx).  
+
 ## Seqera HPC compute environment
 
 To create a new **HPC** compute environment:
 
 1.  In a Seqera workspace, select **Compute environments > New environment**.
-2.  Enter a descriptive name for this environment. Use only alphanumeric characters, dashes, and underscores.
-3.  Select your HPC environment from the **Platform** dropdown menu.
-4.  Select your credentials, or select **+** and **SSH** or **Tower Agent** to add new credentials.
-5.  Enter a name for the credentials.
-6.  Enter the absolute path of the **Work directory** to be used on the cluster.
-7.  Enter the absolute path of the **Launch directory** to be used on the cluster. If omitted, it will be the same as the work directory.
-8.  Enter the **Login hostname**. This is usually the hostname or public IP address of the cluster's login node.
-9.  Enter the **Head queue name**. This is the [default](https://www.nextflow.io/docs/latest/process.html#queue) cluster queue to which the Nextflow job will be submitted.
-10. Enter the **Compute queue name**. This is the [default](https://www.nextflow.io/docs/latest/process.html#queue) cluster queue to which the Nextflow job will submit tasks.
-11. Expand **Staging options** to include optional [pre- or post-run Bash scripts](../launch/advanced.mdx#pre--post-run-scripts) that execute before or after the Nextflow pipeline execution in your environment.
-12. Specify custom **Environment variables** for the head job and/or compute jobs.
-13. Configure any advanced options needed:
+1.  Enter a descriptive name for this environment. Use only alphanumeric characters, dashes, and underscores.
+1.  Select your HPC environment from the **Platform** dropdown menu.
+1.  Select your existing managed identity, SSH, or Tower Agent credentials, or select **+** and **SSH** or **Tower Agent** to add new credentials.
+1.  Enter the absolute path of the **Work directory** to be used on the cluster.
+
+  :::caution 
+  All managed identity users must be a part of the same Linux user group. The group must have access to the HPC compute environment work directory. Set group permissions for the work directory as follows (replace `sharedgroupname` and `<WORKDIR>` with your group name and work directory):
+
+  ```bash
+  chgrp -R sharedgroupname <WORKDIR>
+  chmod -R g+wxs <WORKDIR>
+  setfacl -Rdm g:sharedgroupname:rwxs <WORKDIR>
```

Comment:
Unless I'm missing something the `setfacl` command is still wrong, it should be 
> setfacl -Rdm g::rwX

no `s` flag and capital `X` instead of `x`